### PR TITLE
[DESKTOP] Atomic Design Migration

### PR DESCRIPTION
**Pull request by jvnm-dev**
Migration to the Atomic Design Pattern
See https://github.com/diegohaz/arc/wiki/Atomic-Design

### CODE REVIEW

**Comment by jvnm-dev** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>API changes but no tests!</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Desktop changes but no cypress tests!</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>You should add an entry into CHANGELOG.md!</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Hi there, thanks for contributing!</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bd3db5d3be75981554c9b31b42fc3d836699599e
</p>

File: projects/desktop/src/components/atoms/common/languageChooser/LanguageListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+import { ListItem } from '@material-ui/core'
+
+/**
+ * This component is the list item in the language chooser
+ * Each item is display on a line and centered horizontally
```
Comment (Sylv11) [inline projects/desktop/src/components/atoms/common/languageChooser/LanguageListItem.tsx]: displayed

File: projects/desktop/src/components/atoms/singleForm/input/Input.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { InputAdornment, IconButton } from '@material-ui/core'
 import { VisibilityOff, Visibility } from '@material-ui/icons'
 import { useTranslation, UseTranslationResponse } from 'react-i18next'
 
-import { InputContainer, InputBorder, InputField, InputWithError, InputError } from '../index'
+import { InputContainer, InputBorder, InputField, InputWithError, InputError } from '../../../'
```
Comment (Sylv11) [inline projects/desktop/src/components/atoms/singleForm/input/Input.tsx:6]: Is that normal?

File: projects/desktop/src/components/molecules/singleForm/fields/SFTextField.tsx [hunk 3]
```diff
@@ -2,25 +2,32 @@ import React from 'react'
 import { useTranslation, UseTranslationResponse } from 'react-i18next'
 import { Field } from 'react-final-form'
 
-import { composeValidators, required, validateMinimumLength } from '../../../constants/validators'
-import { Input } from '../../login/form'
+import { composeValidators } from '../../../../constants/validators'
```
Comment (Sylv11) [inline projects/desktop/src/components/molecules/singleForm/fields/SFTextField.tsx:5]: There is a space between imports here?

Comment (Sylv11) [reply projects/desktop/src/components/molecules/singleForm/fields/SFTextField.tsx:5]: Oh no I don't think so

Comment (jvnm-dev) [reply projects/desktop/src/components/molecules/singleForm/fields/SFTextField.tsx:5]: Yes, there is a space

File: projects/desktop/src/components/atoms/singleForm/input/Input.tsx (hunk 2 above)
Comment (jvnm-dev) [reply projects/desktop/src/components/atoms/singleForm/input/Input.tsx:6]: Yes

**Pull request merged**